collectors/python.d.plugin/postgres/postgres.chart.py

```diff
@@ -650,6 +662,43 @@
 """,
 }
 
+QUERY_EMERGENCY_AUTOVACUUM = {
+    DEFAULT: """
```

Comment:
Since you've added only `DEFAULT`, is this query valid for all versions?

Reply:
Hi @ilyam8, I verified these changes work against 9.6, 10, 11, 12, and 13.